### DISCUSSION TITLE
any example of ESP32-internal-adc -> bluetooth?

### DISCUSSION THREAD

**Discussion by aguaviva** [pschatzmann/arduino-audio-tools#223] [Q&A] [answered]
I'd love to connect the ESP32 to a electric guitar and hear the sound in a bluetooth speaker :)  

Thanks!

**Comment by pschatzmann** [comment, marked as answer]
With Bluetooth, the latency is too bad to be useful. I think it is around half a second...

The simplest way to connect a Guitar to an ESP32 is to use a cheap Guitar Pocket Headphone Amplifier that you plug into the line in of an [audiokit](https://github.com/pschatzmann/arduino-audiokit).

**Reply by aguaviva** [reply]
Wow, I didn't know about those amplifiers, thanks!

As for the BT taking half a second, when I watch a video on my phone and hear it on my BT speakers I see no delay, what makes the ESP32 be so slow?
